packages/discord.js/src/structures/GuildMember.js

```diff
@@ -210,6 +210,15 @@ class GuildMember extends Base {
     return this.user.id;
   }
 
+  /**
+   * The DM between the client's user and this member
```

Comment:
Might be worth mentioning that it only returns the channel if it is cached (I know the description is the same as the User class and the private method, but perhaps those should be updated too)

Reply:
I don't know. One could assume it returns `null` if the channel does not exist at all

Reply:
hmm not sure, it's still a getter and all getters rely on cache (or almost all) so I don't think it's necessary. If a maintainer does I can change this and the other descriptions so I'll keep this open until one of them resolves it